Nos/Views/ProfileHeader.swift

```diff
@@ -74,6 +76,11 @@ struct ProfileHeader: View {
                         } label: {
                             Text("\(Localized.following.string): \(author.follows?.count ?? 0)")
                         }
+                        
+                        Spacer()
+                        Text("\(Localized.uns.string): \(verifiedNip05Identifier)")
```

Comment:
This variable starts out as an empty string, and will only be updated after the identifier has been verified.